### PR TITLE
Update in help file template

### PR DESCRIPTION
**Pull request by rpitonak**
- More detailed help file, helps user to better understand how to use image.
- ENVIROMENT VARIABLES heading to specify how to run the image in different ways without the need of rebuilding image.

Example help file for Perl s2i image: https://raw.githubusercontent.com/container-images/perl/master/root/help.md

### CODE REVIEW

File: creating/help.adoc [hunk 1]
```diff
@@ -20,27 +20,31 @@ The help file must be located in the images as /help.1 and in 'man' format.
...
 The following headings are strongly encouraged in the help file for an image.
 
+===== NAME
+Image name with short description.
+
 ===== DESCRIPTION
-Describe in greater detail the role or purpose of the image.  This can include more specifics about the
-packages that make up the image.  You can also describe whether image is meant to be interactive
-or more service oriented.
+Describe how image is used (user app, service, base image, builder image, etc.), the services or features it provides, and environment it is intended to run in (stand-alone docker, atomic super-privileged, oc multi-container app, etc.).
```
Comment (TomasTomecek) [inline creating/help.adoc]: Can we stick to the former description?

I would incorporate your changes like this:

    Describe in greater detail the role or purpose of the image (application, service, base image, builder image, etc.).

I'm not sure how to incorporate the sentence about environment (is it even relevant?).

File: creating/help.adoc [hunk 2]
```diff
@@ -20,27 +20,31 @@ The help file must be located in the images as /help.1 and in 'man' format.
...
 ==== Optional headings
 
 Use the following sections in your help file when applicable.
 
-===== Labels
-Describe the labels you use and why when applicable.  Specifically focus any action-oriented labels like RUN, INSTALL, and
-UNINSTALL.
+===== LABELS
+Describe LABEL settings (from the Dockerfile that created the image) that contains pertinent information.
+For containers run by atomic, that could include INSTALL, RUN, UNINSTALL, and UPDATE LABELS. Others could
```
Comment (TomasTomecek) [inline creating/help.adoc]: I'm not sure if it's relevant to describe labels such as name, version, release...

File: creating/help.adoc [hunk 3]
```diff
@@ -20,27 +20,31 @@ The help file must be located in the images as /help.1 and in 'man' format.
...
+include BZComponent, Name, Version, Release, and Architecture.
 
-===== Security Implications
+===== SECURITY IMPLICATIONS
 If you image uses any privileges that you want to make the user aware of, be sure to document which ones are used and optionally
```
Comment (eliskasl) [inline creating/help.adoc]: If your image...

File: creating/help.adoc [hunk 4]
```diff
@@ -20,27 +20,31 @@ The help file must be located in the images as /help.1 and in 'man' format.
...
 
 ==== Optional headings
 
 Use the following sections in your help file when applicable.
 
-===== Labels
-Describe the labels you use and why when applicable.  Specifically focus any action-oriented labels like RUN, INSTALL, and
-UNINSTALL.
+===== LABELS
+Describe LABEL settings (from the Dockerfile that created the image) that contains pertinent information.
```
Comment (eliskasl) [inline creating/help.adoc]: ...that contain...

File: creating/help.adoc [hunk 5]
```diff
@@ -53,47 +57,62 @@ in multiple ways.
...
 
-  atomic run XYZimage
+    # atomic run XYZimage
 
 To remove the XYZ container (not the image) from your system, run:
 
-  atomic uninstall XYZimage
+    # atomic uninstall XYZimage
+
+Also, describe default configuration options (when defined): hostname, domainname, user ID run as, exposed ports, volumes, working directory, command run by default, etc.
```
Comment (TomasTomecek) [inline creating/help.adoc]: hostname, domain name are irrelevant, right? I would change "user ID run as" to "default user", same for "command run by default" -- "default command".

Comment (eliskasl) [inline creating/help.adoc]: ...the default...

File: creating/help.adoc [hunk 6]
```diff
@@ -53,47 +57,62 @@ in multiple ways.
...
 
-To upgrade the XYZ container from your system, run:
+# ENVIRONMENT VARIABLES
+Explain all environment variables available to run the image in different ways without the need of rebuilding the image. Change variables on the docker command line with -e option. For example:
```
Comment (eliskasl) [inline creating/help.adoc]: ...all the...

File: creating/help.adoc [hunk 7]
```diff
@@ -53,47 +57,62 @@ in multiple ways.
...
 Describe LABEL settings (from the Dockerfile that created the image) that contains pertinent information.
 For containers run by atomic, that could include INSTALL, RUN, UNINSTALL, and UPDATE LABELS. Others could
-include BZComponent, Name, Version, Release, and Architecture.
+include BZComponent, Name, Version, Release, and Architecture. This section is optional.
+
 
 # SECURITY IMPLICATIONS
-If you expose ports or run with privileges, it would be warranted to briefly note those and provide
-an explanation if needed.
+If you expose ports or run with privileges, note those and provide an explanation. For example:
```
Comment (TomasTomecek) [inline creating/help.adoc:99]: can we get better examples here? E.g. postfix requires root, dhcp-client requires capability to manage networking.

File: creating/help.adoc [hunk 8]
```diff
@@ -53,47 +57,62 @@ in multiple ways.
...
+
+-d
+    Runs continuously as a daemon process in the background
+
+-p 3306:3306
+    Opens  container  port  3306  and  maps it to the same port on the Host.
```
Comment (eliskasl) [inline creating/help.adoc]: A few duplicated spaces between words.

File: creating/help.adoc [hunk 9]
```diff
@@ -53,47 +57,62 @@ in multiple ways.
...
+
 
 # HISTORY
 Similar to a Changelog of sorts which can be as detailed as the maintainer wishes.
+
+# SEE ALSO
+
+Does Red Hat provide MariaDB technical support on RHEL 7? https://access.redhat.com/solutions/1247193
```
Comment (TomasTomecek) [inline creating/help.adoc]: I would also point to upstream documentation (git repository of some image on github).

**Review by eliskasl — COMMENTED** [review]
Just a few grammar corrections :) Otherwise looks good!

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
I'm not sure about guidelines and writing style, I just provided comments to content.

File: creating/help.adoc (hunk 4 above)
Comment (eliskasl) [reply creating/help.adoc]: One more thing, a link to the label repo could be useful: https://github.com/projectatomic/ContainerApplicationGenericLabels/tree/master/vendor/redhat

**Comment by rpitonak** [comment]
Updated, please have another look.

File: creating/help.adoc [hunk 10]
```diff
@@ -53,47 +57,67 @@ in multiple ways.
...
 
 # SECURITY IMPLICATIONS
-If you expose ports or run with privileges, it would be warranted to briefly note those and provide
-an explanation if needed.
+If you expose ports or run with privileges, note those and provide an explanation. For example:
+
+Root privileges
+    Container is running as root. Explain why is it necessary.
+
+-d
```
Comment (TomasTomecek) [inline creating/help.adoc]: Can we remove this item? I don't understand how it is a security concern.

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
Just one nit, otherwise looks good. Nice work!

**Pull request merged**